src/components/NavBar.tsx

```diff
@@ -153,18 +153,25 @@ export default function NavBar({
         </SubNavbar>
       </NavItem>
       {trackingEnabled && (
-        <>
-          <NavSection title={strings.PLANTS.toUpperCase()} />
-          <NavItem
-            label={strings.DASHBOARD}
-            icon='iconRestorationSite'
-            selected={!!isPlantsDashboardRoute}
-            onClick={() => {
-              closeAndNavigateTo(APP_PATHS.PLANTS_DASHBOARD);
-            }}
-            id='plants-dashboard'
-          />
-        </>
+        <NavItem
+          label={strings.PLANTS}
+          icon='iconRestorationSite'
+          id='plants'
+          onClick={() => {
+            closeAndNavigateTo(APP_PATHS.PLANTS_DASHBOARD);
+          }}
+        >
```

Comment:
Alternatively, I could have made this change:

```suggestion
        <NavItem
          label={`${strings.PLANTS.charAt(0).toUpperCase()}${strings.PLANTS.slice(1).toLowerCase()}`}
          icon='iconRestorationSite'
          id='plants'
          onClick={() => {
            closeAndNavigateTo(APP_PATHS.PLANTS_DASHBOARD);
          }}
        >
```